### ISSUE TITLE
Incorrect routes through the Northwest Pacific

### ISSUE THREAD

**Issue by pjwilliams11** [genthalili/searoute-py#20]
This is an awesome and powerful package! One thing I've found: Routes through the Northwest Pacific are erroneous. It seems the base graph lacks sufficient points in the Northwest pacific to properly calculate a minimal distance route, leading to some weird routing. See the image below for a route between Shanghai and Vancouver - this should go north of Japan and generally follow a great circle route across the rest of the Pacific.

![Screenshot 2023-08-29 at 2 13 23 PM](https://github.com/genthalili/searoute-py/assets/124653521/54257c3f-a1be-42da-8725-1a65321eb4b8)

**Comment by genthalili** [comment]
Hi @pjwilliams11, 
You're right, looks like the network is not dense enough to propose the shortest path as expected:
![image](https://github.com/genthalili/searoute-py/assets/3913876/37f9d54a-8470-4939-83a2-98ac0bc71dd5)

When I find some free time, I'll look closely to this case and see how to fix it as there must be connection secured between west and east properly with the "segment" file.

Anyways, meanwhile feel free to contribute/send pull request.

**Comment by vinisalazar** [comment]
Hi @genthalili, I am experiencing a similar problem. I have these two points:

```python
p0, p1 = array([-154.9234, 71.9052], dtype=object), array([-72.46, 72.23], dtype=object)
```

I am setting `restrictions=None` to allow routes through the Northwestern passages:

```python
dist_ = sr.searoute(p0, p1, restrictions=None)
```

This is my resulting route:

```python
plot_world_map_with_lines([dist_["geometry"]["coordinates"],])
```
![searoute](https://github.com/genthalili/searoute-py/assets/17276653/9136663d-b1ff-467d-92e6-6cb622111d1e)

I would like instead to have a shorter distance through the Northwestern passages. This superestimation of the distance seems to happen with many of my data points around the poles. Is there to maybe obtain a denser network?

Thank you for your assistance and for providing Searoute!

**Comment by genthalili** [comment]
Just released version `1.3.1` which should cover both issues.
Try it and let me know.

**Comment by genthalili** [comment]
closing this as no activity, feel free to re-open this issue if needed
